episodes/4-advanced-layer-types.Rmd

```diff
@@ -27,45 +27,35 @@ So far, we have seen one type of layer, namely the **fully connected**, or **den
-When loading the CIFAR-10 dataset, you might get the following error:
-```
-[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1125)
-```
-You can solve this error by adding this to your notebook:
-```python
-import ssl
-ssl._create_default_https_context = ssl._create_unverified_context
-```
-:::
-
-## CIFAR-10
-
-The CIFAR-10 dataset consists of images of 10 different classes: airplanes, cars, birds, cats, deer, dogs, frogs, horses, ships, and trucks.
-It is widely used as a benchmark dataset for image classification. The low resolution of the images in the dataset allows for quick loading and testing models.
-
-For more information about this dataset and how it was collected you can check out
-[Learning Multiple Layers of Features from Tiny Images by  Alex Krizhevsky, 2009](https://www.cs.toronto.edu/~kriz/learning-features-2009-TR.pdf).
+The MLCommons Dollar Street Dataset is a collection of images of everyday household items from homes around the world that visually captures socioeconomic diversity of traditionally underrepresented populations. We use a subset of the original dataset that can be used for multiclass classification with 10 categories. Let's load the data:
 
-
-![Sample images from the CIFAR-10 data-set. Each image is labelled with a category, for example: 'frog' or 'horse'](fig/04_cifar10.png){alt="A 5 by 5 grid of 25 sample images from the CIFAR-10 data-set. Each image is labelled with a category, for example: 'frog' or 'horse'."}
-
-We take a small sample of the data as training set for demonstration purposes.
 ```python
-n = 5000
-train_images = train_images[:n]
-train_labels = train_labels[:n]
-```
-
-
+import pathlib
+DATA_FOLDER = pathlib.Path('data/dataset_dollarstreet/') # change to location where you stored the data
+train_images = np.load(DATA_FOLDER / 'train_images.npy')
+val_images = np.load(DATA_FOLDER / 'test_images.npy')
+train_labels = np.load(DATA_FOLDER / 'train_labels.npy')
+val_labels = np.load(DATA_FOLDER / 'test_labels.npy')
+```
+
+## Dollar Street 10 dataset
+
+The dollar street 10 dataset consists of images of 10 different classes, this is the mapping of the categories:
+| Category        | label |
+|-----------------|-------|
+| day bed         | 0     |
+| dishrag         | 1     |
+| plate           | 2     |
+| running shoe    | 3     |
+| soap dispenser  | 4     |
+| street sign     | 5     |
+| table lamp      | 6     |
+| tile roof       | 7     |
+| toilet seat     | 8     |
+| washing machine | 9     |
+
+
+![Sample images from the dollar street 10 data-set. Each image is labelled with a category, for example: 'street sign' or 'soap dispenser'](fig/04_dollar_street_10){alt="A 5 by 5 grid of 25 sample images from the dollar street 10 data-set. Each image is labelled with a category, for example: 'street sign' or 'soap dispenser'."}
 ## 2. Identify inputs and outputs
 
 ## Explore the data
```

Comment:
```suggestion
### Explore the data
```

I think this and some below should be third-level headings, to present the heading hierarchy correctly.